tests/format/scss/parens/__snapshots__/format.test.js.snap

```diff
@@ -29,7 +29,8 @@ $icons: (
     top: 73,
   ),
 
-  /* Should preserve empty lines */ cal-week-group:
+  /* Should preserve empty lines */
+    cal-week-group:
```

Comment:
I'm not sure whether it get correct or incorrect yet.

Reply:
Maybe dedented one looks fine but I can't come up with good heuristic and good doc structure.

Reply:
There should be a `line` before, maybe we can find it and replace with `softline`?

Reply:
Don't you mistake - for + in diff? Or do I misunderstand?

Reply:
replacing by dedent(line) might work well here but I'm worrying to break somewhere else.

Reply:
Do this makes sense to you?
https://github.com/prettier/prettier/pull/16570/commits/54f924df951b1b22389a10ea8a301647a1839591